builder/image/bin/boot

```diff
@@ -66,16 +66,16 @@ DOCKER_PID=$!
 
 # wait for docker to start
 while [[ ! -e /var/run/docker.sock ]]; do
-  sleep 1
+	sleep 1
 done
 
 # HACK: load progrium/cedarish tarball for faster boot times
 # see https://github.com/deis/deis/issues/1027
 if ! docker history progrium/cedarish >/dev/null 2>/dev/null ; then
-    echo "Loading cedarish..."
-    cat /progrium_cedarish.tar.gz | docker import - progrium/cedarish
+		echo "Loading cedarish..."
+		cat /progrium_cedarish.tar.gz | docker import - progrium/cedarish
 else
-    echo "Cedarish already loaded"
+		echo "Cedarish already loaded"
```

Comment:
Well even if we don't have a clear standard for shell indentation, this kind of change just makes it harder to review the overall diff and should be omitted IMHO.